apps/core/scenes/search/SearchContext.js

```diff
@@ -26,12 +26,13 @@ export type Location = {|
 |};
 
 export type LocationSearchType = 'travelTo' | 'travelFrom';
-
+export type SortTypes = 'QUALITY' | 'PRICE' | 'DURATION';
 type State = {|
   tripType: TripTypes,
   travelFrom: Array<Location>,
   travelTo: Array<Location>,
   dateFrom: Date,
+  sort: SortTypes,
```

Comment:
could this be named `sortBy` instead?

Reply:
@tbergq sure!! Should we also change it on the GraphQL schema?

Reply:
🤔  Maybe, but probably not within this PR.